TalismanSociety/signet-web
# [FEAT] - NeuroWeb testnet

# Description

- Depends on https://github.com/TalismanSociety/sigsquid/pull/2

## ⛑️ **What was done:**

Added NeuroWeb testnet.

## Type of change

Please delete options that are not relevant.


### 🧪 **How to test:**

- Request NeuroWeb testnet tokens by following their [Slightly outdated docs](https://docs.origintrail.io/decentralized-knowledge-graph/node-setup-instructions/useful-resources/dkg-testnet-faucet#request-neuroweb-trac-and-neuro). Join their [Discord](https://discord.com/invite/gYq6GuJ4sJ) and post `!fundme_otp <substrate_wallet_address>` on the testnet-discussion channel.
- Go to polkadot multisig
- Create a Multisig Vault
- Create Drafts, sign and Execute Transactions

Expected Result:
- Draft txs should be displayed on the Draft tab
- Pending signature txs should be displayed on the Pending tab
- Execute tx should be displayed on the History tab 

### **Note:**

Executed tx will **NOT** be displayed on history until https://github.com/TalismanSociety/sigsquid/pull/2 is merged.

### 🖼️ **Screenshots:**

#### Desktop:

![Screenshot 2024-04-30 at 16 29 01](https://github.com/TalismanSociety/signet-web/assets/47801291/70eb65e7-021a-4a70-9776-2eeedad24e30)
![Screenshot 2024-04-30 at 16 29 07](https://github.com/TalismanSociety/signet-web/assets/47801291/1bfa6ca8-168e-411a-8409-51b19ed5a343)
